tests/test_diff_tally/test_diff_tally.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+
+import glob
+import os
+try:
+    from StringIO import StringIO
+except:
+    from io import StringIO
```

Comment:
Yes, looks like [io.StringIO is unicode-aware](http://stackoverflow.com/a/3410461/1572453). The solution would be to decode the str returned by `df.to_csv`, i.e., `df.to_csv().decode()`.